src/modules/rlm_lua/configure.ac

```diff
@@ -7,6 +7,9 @@ m4_include([m4/ax_lua.m4])
 if test x$with_[]modname != xno; then
 	AC_PROG_CC
 	AC_PROG_CPP
+	AC_PROG_SED
+
+	LUA_VERSION="5.2"
```

Comment:
We still prefer luajit-2.0

Reply:
I'm not sure what the intention of these changes are? What does this fix?

Reply:
Because the current configure.ac does not detect the luakit as well.

```
[root@docker-freeradius rlm_lua]# dpkg -l | grep luajit
ii  libluajit-5.1-2:amd64            2.1.0~beta3+dfsg-5.1              amd64        Just in time compiler for Lua - library version
ii  libluajit-5.1-common             2.1.0~beta3+dfsg-5.1              all          Just in time compiler for Lua - common files
ii  libluajit-5.1-dev:amd64          2.1.0~beta3+dfsg-5.1              amd64        Just in time compiler for Lua - development files
[root@docker-freeradius rlm_lua]# ./configure 1> /dev/null 2>&1
[root@docker-freeradius rlm_lua]# pwd
/root/Devel/FreeRadius/freeradius-server.git/src/modules/rlm_lua
[root@docker-freeradius rlm_lua]# grep _lua config.log
configure:4467: WARNING: silently not building rlm_lua.
configure:4469: WARNING: FAILURE: rlm_lua requires: lua.h.
config.status:804: creating rlm_lua.mk
ac_cv_header_lua_h=no
ac_cv_header_luaconf_h=no
ac_cv_header_luajit_h=no
ac_cv_header_lualib_h=no
[root@docker-freeradius rlm_lua]#
```

Reply:
The reason of that patch is because the current configure.ac do not detect Lua (only Luajit) as can be seen below.

```
[root@docker-freeradius freeradius-server.git]# dpkg -l | grep lua
ii  liblua5.2-0:amd64                5.2.4-1.1build1                   amd64        Shared library for the Lua interpreter version 5.2
ii  liblua5.2-dev:amd64              5.2.4-1.1build1                   amd64        Development files for the Lua language version 5.2
ii  libluajit-5.1-2:amd64            2.1.0~beta3+dfsg-5.1              amd64        Just in time compiler for Lua - library version
ii  libluajit-5.1-common             2.1.0~beta3+dfsg-5.1              all          Just in time compiler for Lua - common files
ii  libluajit-5.1-dev:amd64          2.1.0~beta3+dfsg-5.1              amd64        Just in time compiler for Lua - development files
ii  lua5.2                           5.2.4-1.1build1                   amd64        Simple, extensible, embeddable programming language
ii  luajit                           2.1.0~beta3+dfsg-5.1              amd64        Just in time compiler for Lua programming language version 5.1
[root@docker-freeradius freeradius-server.git]# 

```

```
[root@docker-freeradius freeradius-server.git]# git clean -fdx
[root@docker-freeradius freeradius-server.git]# ./configure --with-rlm-lua 1> /dev/null 2>&1
[root@docker-freeradius freeradius-server.git]# make
[root@docker-freeradius freeradius-server.git]# find . -iname rlm_lua.so
[root@docker-freeradius freeradius-server.git]#
```